src/clarity-angular/tabs/vtabs-nav-group.ts

```diff
@@ -0,0 +1,25 @@
+
+import {Component} from "@angular/core";
+
+/**
+ * @class VTabsNavGroup
+ *
+ * @description
+ * A child component of VTabs that is a container for a group of nav links.
+ * An optional label can be passed in to give the group a title.
+ * clr-vtabs-nav-links are projected to the VTabsNavLink class.
+ *
+ * Style is set in the template, not dynamically.
+ */
+@Component({
+    selector: "clr-vtabs-nav-group",
+    template: `
+        <div class="nav-group">
+            <ng-content select="label"></ng-content>
```

Comment:
@hippee-lee: Actually you can ignore my comment about the dropdowns. I looked and the issue was dropdown requiring an attribute selector was different. You can definitely use css selectors like @mathisscott mentioned.